analysis/common_variables.py

```diff
@@ -41,7 +41,7 @@ def generate_common_variables(index_date_variable):
         return_expectations={
```

Comment:
I think this should be between index data and end data ? rather than on or after index date ?

Reply:
Sorry about that, I meant to add it on on_or_after! If it is fine with you it's ok. But in [my code](https://github.com/opensafely/post-covid-gastrointestinal/blob/main/analysis/common_variables.py)  I've been told to change it to between for the exposure variables.

Reply:
It's fine with me as it makes no difference  - for your code it makes sense to make it as robust as possible as it will be used for multiple projects but I'm fine with leaving it as it is as I'd need to update for all the outcomes as well for consistency and it won't make a difference